src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props

```diff
@@ -3,6 +3,32 @@
     Properties and tasks supporting Razor MSBuild integration
   -->
 
+  <!--
+    Default properties for common Razor SDK behavior.
+  -->
+  <PropertyGroup>
+    <!--
+      Set to true to automatically include Razor (.cshtml) files in @(RazorGenerate) from @(Content).
+    -->
+    <EnableDefaultRazorGenerateItems>true</EnableDefaultRazorGenerateItems>
```

Comment:
This should have a condition on it so you don't override this property if it's already set.

Reply:
Sure